### PR TITLE
Include Microsoft.VisualStudio.CodingConventions in the Code Style packages

### PR DESCRIPTION
**Pull request by sharwell**
This was unintentionally removed in #31276.

### CODE REVIEW

File: src/CodeStyle/VisualBasic/CodeFixes/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes.vbproj [hunk 1]
```diff
@@ -29,6 +29,9 @@
       <_File Include="$(ArtifactsConfigurationDir)Dlls\Microsoft.CodeAnalysis.CodeStyle\**\Microsoft.CodeAnalysis.CodeStyle.resources.dll"/>
       <_File Include="$(ArtifactsConfigurationDir)Dlls\Microsoft.CodeAnalysis.CodeStyle.Fixes\**\Microsoft.CodeAnalysis.CodeStyle.Fixes.resources.dll"/>
 
+      <!-- .editorconfig support prior to new support in the compiler -->
+      <_File Include="$(ArtifactsConfigurationDir)Dlls\Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes\Microsoft.VisualStudio.CodingConventions.dll"/>
```
Comment (JoeRobich) [inline src/CodeStyle/VisualBasic/CodeFixes/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes.vbproj]: How does this assembly make it into the build folder for this project? I would have expected a package reference for CodingConventions. What am I missing?

Comment (sharwell) [reply src/CodeStyle/VisualBasic/CodeFixes/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes.vbproj]: The package is included via this:

https://github.com/dotnet/roslyn/blob/5574867382e155ed1e2ed1054ef44234d03d1619/src/CodeStyle/Core/Analyzers/Microsoft.CodeAnalysis.CodeStyle.csproj#L14

It's copied to the output folder via this:

https://github.com/dotnet/roslyn/blob/5574867382e155ed1e2ed1054ef44234d03d1619/src/CodeStyle/CSharp/CodeFixes/Microsoft.CodeAnalysis.CSharp.CodeStyle.Fixes.csproj#L10

Comment (sharwell) [reply src/CodeStyle/VisualBasic/CodeFixes/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes.vbproj]: ➡️ It turns out the VB packaging was not correct (it omitted the `CopyLocalLockFileDependencies` flag). I've corrected this now.

Comment (JoeRobich) [reply src/CodeStyle/VisualBasic/CodeFixes/Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes.vbproj]: Thanks! I've learned something new today

**Pull request merged**